reset-virtual-host.cmd

```diff
@@ -1,35 +1,4 @@
-:: the windows shell, so amazing
-
-:: options
 @echo Off
 cd %~dp0
-setlocal
-
-:: determine dirs
-set NUGET_CACHE_DIR=%LocalAppData%\.nuget\v4.1.0
-set NUGET_LOCAL_DIR=.nuget\v4.1.0
-
-:: download nuget to cache dir
-set NUGET_URL=https://dist.nuget.org/win-x86-commandline/v4.1.0/nuget.exe
-if not exist %NUGET_CACHE_DIR%\NuGet.exe (
-  if not exist %NUGET_CACHE_DIR% md %NUGET_CACHE_DIR%
-  echo Downloading '%NUGET_URL%'' to '%NUGET_CACHE_DIR%\NuGet.exe'...
-  @powershell -NoProfile -ExecutionPolicy unrestricted -Command "$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest '%NUGET_URL%' -OutFile '%NUGET_CACHE_DIR%\NuGet.exe'"
-)
-
-:: copy nuget locally
-if not exist %NUGET_LOCAL_DIR% md %NUGET_LOCAL_DIR%
-if not exist %NUGET_LOCAL_DIR%\NuGet.exe (
-  copy %NUGET_CACHE_DIR%\NuGet.exe %NUGET_LOCAL_DIR%\NuGet.exe > nul
-)
-
-:: restore packages
-%NUGET_LOCAL_DIR%\NuGet.exe restore .\packages.config -PackagesDirectory ./script-packages -MSBuildVersion 15 -Verbosity quiet
-
-:: find VS install path
-for /f "usebackq tokens=*" %%i in (`script-packages\vswhere.1.0.62\tools\vswhere -latest -products * -requires Microsoft.Component.MSBuild -property installationPath`) do (
-  set VS_INSTALL_PATH=%%i
-)
 
-:: run script
-"%VS_INSTALL_PATH%\MSBuild\15.0\Bin\Roslyn\csi.exe" reset-virtual-host.csx %*
\ No newline at end of file
+dotnet run --project src/targets --no-launch-profile -- %*
```

Comment:
What's the need for `--no-launch-profile`?

Reply:
@bording I usually have that in there so that if I'm ever debugging the targets project in VS, and supplying arguments to it, it will not affect running the command in the console. However, you can argue YAGNI, so I'm happy to remove it.